game-core/src/main/java/org/triplea/game/server/HeadlessGameServer.java

```diff
@@ -109,21 +104,13 @@ private HeadlessGameServer() {
 
   @SuppressWarnings("FutureReturnValueIgnored") // false positive; see https://github.com/google/error-prone/issues/883
   private void startLobbyWatcher() {
-    int reconnect;
-    try {
-      final String reconnectionSeconds = System.getProperty(LOBBY_GAME_RECONNECTION,
-          "" + LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT);
-      reconnect = Math.max(Integer.parseInt(reconnectionSeconds), LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM);
-    } catch (final NumberFormatException e) {
-      reconnect = LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT;
-    }
     lobbyWatcherResetupThread.scheduleAtFixedRate(() -> {
       try {
         restartLobbyWatcher();
       } catch (final Exception e) {
         log.log(Level.WARNING, "Failed to restart Lobby watcher", e);
       }
-    }, reconnect, reconnect, TimeUnit.SECONDS);
+    }, LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT, LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT, TimeUnit.SECONDS);
```

Comment:
Just a minor thing I noticed:
Instead of storing the amount of seconds within 12 hours, it would make more sense to store the 12 hours and use `TimeUnit.HOURS` in this line instead.

Reply:
I like the idea; I'm reluctant to change the scope of this branch. Did we find out if 12+ hours is correct?

Reply:
Excellent context @ssoloff , thank you. Did you run across per-chance the reason why the disconnect was added?

Reply:
Following up, in prod we set this value to 2 days. I've updated this branch to match: c45d87517

I've opened issue: https://github.com/triplea-game/triplea/issues/4552 to discuss what we would like to do long-term with this property and if it is correct or not: "lobby.game.reconnection time does not seem right"

Reply:
> Did you run across per-chance the reason why the disconnect was added?

Unfortunately, no (or, at least, I didn't recognize it as such if I did come across it).